### DISCUSSION TITLE
Download Fails

### DISCUSSION THREAD

**Discussion by Bender8** [gilesknap/gphotos-sync#474] [Q&A]
I am having an issue with files failing to download. Seems like I get between 10 and 20 photos before I get a couple of errors and the program stops.  See attached snapshot of the error message.

Any help would be greatly appreciated.
![Screenshot_20240212_214510](https://github.com/gilesknap/gphotos-sync/assets/159867740/32e6097a-85fd-4b36-b439-abbd4dbfef3f)

**Comment by Bender8** [comment]
Ran it again today. The process completed without failure. Not sure what was wrong before, suspect it was being throttled on Google's end.

**Reply by gilesknap** [reply]
Yep I think that looks like a transient server issue.
